applications/nrf_desktop/README.rst

```diff
@@ -378,34 +378,39 @@ nRF Desktop build types
 The nRF Desktop does not use a single :file:`prj.conf` file.
 Configuration files are provided for different build types for each supported board.
 
+The :file:`prj.conf` represents a ``debug`` build type and is available for each board.
+Each other build types has a dedicated file with name following a pattern :file:`prj_${BUILD_TYPE}.conf`.
+
+In case there are other configuration files associated with partition layout or child image configuration, these follow the same pattern with build type added to the file name as a suffix.
+
 .. include:: /gs_modifying.rst
    :start-after: build_types_overview_start
    :end-before: build_types_overview_end
```

Comment:
These lines describe what a build type is. They need to follow the first paragraph. Move them to line 381.